tests/conftest.py

```diff
@@ -18,6 +18,21 @@
 yaml.add_multi_constructor("!", lambda loader, suffix, node: None, Loader=yaml.SafeLoader)
 
 
+@pytest.fixture
+def hyphenated_application_name() -> str:
+    return "hyphenated-application-name"
+
+
+@pytest.fixture
+def alphanumeric_environment_name() -> str:
+    return "alphanumericenvironmentname123"
+
+
+@pytest.fixture
+def alphanumeric_service_name() -> str:
+    return "alphanumericservicename123"
```

Comment:
Created these as fixtures so it's easier to change later if we decide we want different behaviour.

Reply:
It'd make more sense to just put some constants at the top of the file.

Reply:
I had wondered about that myself. What's our criteria for this? If it's a scalar type use a constant, more complicated use a fixture?

Reply:
@WillGibson That makes sense to me. I think these would be constants in this file and if used elsewhere, we put them in the `conftest.py` file.

Reply:
They are constants now.